main.js

```diff
@@ -88,28 +68,55 @@ class Timer {
 }
 
 class DOM_Manager {
-    constructor() {
-        this.cardsCountVar = document.getElementsByClassName("card").length;
+    constructor(Fn) {
+        this.callBackFn=Fn;
         this.clickListenerBinded = this.clickListener.bind(this);
+        this.submitOnClickBinded = this.submitOnClick.bind(this);
         this.imgs = [];
-        for(let i=0;i<this.cardsCountVar;i++){
-            this.imgs.push(document.getElementById("img"+i));
-        }
+        this.imagesRandomised=[];
+        document.getElementById("Easy").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Medium").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Hard").addEventListener("change", this.difficultOnChange);
+        document.getElementById("submit").addEventListener("click", this.submitOnClickBinded);
+        this.timer=new Timer();
+    }
+    difficultOnChange() {
+        document.getElementById("submit").disabled = false;
+    }
+    submitOnClick() {
+        this.cardsCount = document.getElementById("Easy").checked ? 12 : this.cardsCount;
+        this.cardsCount = document.getElementById("Medium").checked ? 20 : this.cardsCount;
+        this.cardsCount = document.getElementById("Hard").checked ? 30 : this.cardsCount;
+        document.getElementById("myModal").style = "display:none";

```

Comment:
Create `hidden` class in CSS and use it instead of adding inline styles.
Different CSS frameworks use this technique